mava/wrappers/jaxmarl.py

```diff
@@ -287,6 +304,16 @@ def action_mask(self, state: JaxMarlState) -> Array:
             mask = jnp.ones(self._action_shape, dtype=jnp.float32)
         return mask
 
-    def get_global_state(self, obs: Dict[str, Array]) -> Array:
+    def get_global_state(self, state: Array, obs: Dict[str, Array], agents_view: Array) -> Array:
         """Get global state from observation and copy it for each agent."""
-        return jnp.tile(jnp.array(obs["world_state"]), (self._env.num_agents, 1))
+        if hasattr(obs, "world_state"):
+            global_state = jnp.tile(jnp.array(obs["world_state"]), (self._env.num_agents, 1))
+        elif isinstance(self._env, MABraxEnv):
```

Comment:
I wonder if we shouldn't follow a similar approach to Jumanji and have `MABrax` have its own wrapper? Because the isinstance check is not the nicest :sweat_smile: 
Keen to hear your thoughts on this @sash-a?

Reply:
Ye totally agreed! Was actually talking to @liamclarkza about it this morning.

What I'd recommend:
* Have a `JaxMarlBaseWrapper` which keeps all the common functionality
* Have a `SmaxWrapper` and an `MaBraxWrapper` which extends `JaxMarlBaseWrapper` and implements the `_action_shape` and `get_global_shape` methods differently, I think that's all we'd need to do differently :thinking:

Reply:
I didn't think about that it's better for future use as well (if we aim to test other envs) 😅 I'll make the necessary changes then 🙌